### PR TITLE
MOTECH-2094: Added a migration to scheduler module

### PR DESCRIPTION
**Pull request by kmadej**
After scheduler refactor, class service.impl.MotechScheduledJob changed package
to service.MotechScheduledJob. This class name is stored in motechquartz database,
so we lost backward compatibility. This migration should fix the problem and now
old jobs can be properly fired after scheduler API clean up.

### CODE REVIEW

File: modules/scheduler/scheduler/src/main/resources/db/migration/scheduler/default/V3__MOTECH-2094.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (sebbrudzinski) [inline modules/scheduler/scheduler/src/main/resources/db/migration/scheduler/default/V3__MOTECH-2094.sql]: Some comment on why we do that would be in place here.

File: modules/scheduler/scheduler/src/main/resources/db/migration/scheduler/mysql/V3__MOTECH-2094.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (sebbrudzinski) [inline modules/scheduler/scheduler/src/main/resources/db/migration/scheduler/mysql/V3__MOTECH-2094.sql]: Some comment on why we do that would be in place here.

**Comment by sebbrudzinski** [comment]
test this psql

**Comment by sebbrudzinski** [comment]
Doesn't seem to work with PostgreSQL

**Comment by kmadej** [comment]
test this psql

**Pull request merged**